modules/serverless-logic-overview-input-output-schema.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+// * about/serverless-logic-overview.adoc
+
+
+:_content-type: CONCEPT
+[id="serverless-logic-overview-input-output-schema_{context}"]
+= Input and output schema definition for Serverless Logic
```

Comment:
This intial heading is empty without any content. If we have nothing useful to add here, i would suggest to move the Input and output schema definitions content a level above. or maybe we can title the assembly as `Schema definitions ` and build up the context in such a way: SL supports two types of schema definitions ...... and rather than creating two seperate sub-headings, we can label them.